external-import/zerofox/docker-compose.yml

```
@@ -1,14 +1,20 @@
 version: '3'
 services:
   connector-zerofox:
-    image: opencti/connector-zerofox:6.1.0
+    image: opencti/connector-zerofox:latest
     environment:
-      - OPENCTI_URL=http://localhost
-      - OPENCTI_TOKEN=ChangeMe
-      - CONNECTOR_ID=ChangeMe
       - CONNECTOR_NAME=Zerofox
       - CONNECTOR_SCOPE=zerofox # MIME type or Stix Object
-      - CONNECTOR_LOG_LEVEL=error
-      - ZEROFOX_USERNAME=changeme
-      - ZEROFOX_PASSWORD=changeme
+      - OPENCTI_URL=http://opencti-dev-platform:8080
```

I would keep this as localhost
```suggestion
      - OPENCTI_URL=http://localhost
```